docs/script.js

```diff
@@ -6,4 +6,22 @@ window.onload = function() {
 
         document.getElementById('datetime').innerHTML = displayDate + " " + displayTime;
     }, 1000); // 1000 milliseconds = 1 second
-}
\ No newline at end of file
+}
+
+
+/* My new function to check if the name field contain only alphabets */
+function checkName(submitcheck) {
+      var letters = /^[A-Za-z]+$/;
+      if(submitcheck.value.match(letters))
+      {
+      alert('Your name is Valid');
+      console.log('Your name is Valid');
+      return true;
+      }
+      else
+      {
+      alert('Please input alphabet characters only');
+      console.log("Please input alphabet characters only")
+      return false;
+      }
+    }
```

Comment:
This does not actually prevent the form from submitting if the user has entered valid information.  See the sample solution.